packages/core/src/components/Form.tsx

```diff
@@ -793,7 +803,7 @@ export default class Form<
       _set(errorSchemaCopy, path, newErrorSchema);
       newErrorSchema = errorSchemaCopy;
     }
-    if (mustValidate) {
+    if (mustValidate && this.pendingChanges.length === 1) {
```

Comment:
```suggestion
    // If there are pending changes in the queue, skip live validation since it will happen with the last change
    if (mustValidate && this.pendingChanges.length === 1) {
```